generator/handlebars/template/public/kss.css

```diff
@@ -0,0 +1 @@
#kss-node .kss-title__permalink{display:block;color:#000;text-decoration:none}#kss-node .kss-title__permalink:hover,#kss-node .kss-title__permalink:focus,#kss-node .kss-title__permalink:active{color:#0645ad}@media screen and (min-width:607px){#kss-node .kss-title__permalink:hover .kss-title__permalink-hash,#kss-node .kss-title__permalink:focus .kss-title__permalink-hash,#kss-node .kss-title__permalink:active .kss-title__permalink-hash{display:inline}}#kss-node .kss-title__permalink-hash{display:none;color:#ccc}#kss-node .kss-parameters{display:table;list-style-type:none;margin-left:0;padding-left:0}#kss-node .kss-parameters__item{display:table-row}#kss-node .kss-parameters__name{display:table-cell;padding-right:20px}#kss-node .kss-parameters__description{display:table-cell}#kss-node .kss-modifier__wrapper{border:1px solid #ccc;padding:10px}#kss-node .kss-modifier__heading{margin:-10px -10px 10px -10px;padding:10px;border-bottom:1px solid #ccc;background-color:#eee;font-weight:bold}#kss-node .kss-modifier__default-name{font-weight:bold;margin-bottom:12px}#kss-node .kss-modifier__name{float:left;padding-right:10px;font-weight:bold}#kss-node .kss-modifier__description{margin-bottom:12px}#kss-node .kss-modifier__example{clear:left;padding:1px 0;margin:0 0 23px 0}#kss-node .kss-modifier__example:last-child{margin:0}#kss-node .kss-markup{margin:-1px 0 24px 0;border:1px solid #999}#kss-node .kss-markup pre{margin:0}#kss-node .kss-github{display:none}@media screen and (min-width:501px){#kss-node .kss-github{display:block;position:absolute;top:0;right:0}}#kss-node .kss-github img{border:0}#kss-node .kss-modifier__wrapper:-webkit-full-screen,#kss-node .kss-modifier__wrapper:-moz-full-screen,#kss-node .kss-modifier__wrapper:-ms-fullscreen,#kss-node .kss-modifier__wrapper:fullscreen{max-width:initial;width:100vw;min-height:100vh;margin-bottom:0;overflow:scroll;padding:0;border:none}#kss-node .pln{color:#000}#kss-node .str{color:#080}#kss-node .kwd{color:#008}#kss-node .com{color:#800}#kss-node .typ{color:#606}#kss-node .lit{color:#066}#kss-node .pun,#kss-node .opn,#kss-node .clo{color:#660}#kss-node .tag{color:#008}#kss-node .atn{color:#606}#kss-node .atv{color:#080}#kss-node .dec,#kss-node .var{color:#606}#kss-node .fun{color:red}@media print,projection{#kss-node .str{color:#060}#kss-node .kwd{color:#006;font-weight:bold}#kss-node .com{color:#600;font-style:italic}#kss-node .typ{color:#404;font-weight:bold}#kss-node .lit{color:#044}#kss-node .pun,#kss-node .opn,#kss-node .clo{color:#440}#kss-node .tag{color:#006;font-weight:bold}#kss-node .atn{color:#404}#kss-node .atv{color:#060}}#kss-node ol.linenums{margin:0;padding:0 0 3px 0;list-style-type:none}#kss-node ol.linenums li{min-height:24px;border-bottom:1px solid #eee;padding:0 10px;background:#fff}#kss-node ol.linenums li:first-child{padding-top:3px}#kss-node ol.linenums li.L0,#kss-node ol.linenums li.L2,#kss-node ol.linenums li.L4,#kss-node ol.linenums li.L6,#kss-node ol.linenums li.L8{background:#fcfcfc}
```

Comment:
Agree with @RobLoach. Both of these generators/* files look like cruft from rebasing the 2.x work onto the latest 3.x work in master.